Source/Core/VideoCommon/GXPipelineTypes.h

```diff
@@ -39,7 +39,7 @@ struct GXPipelineUid
   GXPipelineUid(const GXPipelineUid& rhs) { std::memcpy(this, &rhs, sizeof(*this)); }
```

Comment:
Same here for memcpy and memset

Reply:
The compiler doesn't actually complain about this one for some reason.